examples/file-upload.html

```diff
@@ -0,0 +1,181 @@
+<title>snuggsi ツ - File Upload</title>
+
+<meta charset=UTF-8>
+<meta name=viewport content=width=device-width,initial-scale=1>
+
+<link rel=stylesheet href=http://www.devpunks.com/typography.css>
+
+<script nomodule    src=https://unpkg.com/snuggsi/snuggsi.js></script>
+<script nomodule    src=https://unpkg.com/snuggsi/examples/webcomponents-hi-ce.js></script>
+
+<hgroup>
+  <h1>snuggsi ツ</h1>
+  <h2>File Upload</h2>
+</hgroup>
+
+<file-upload>
+
+  <input multiple id=files name=files[] type=file>
+  <label for=files ondragover="event.preventDefault()" ondragenter=ondragstart ondragleave=ondragend ondrop=ondragdrop>
```

Comment:
@VicenteRD be careful with that global `event` object. @brandondees taught me doesn't work in FF.